components/eventing-controller/controllers/subscription/nats/reconciler.go

```diff
@@ -416,7 +416,7 @@ func defaultSinkValidator(ctx context.Context, r *Reconciler, subscription *even
 		return fmt.Errorf("sink should contain 5 sub-domains: %s", trimmedHost)
 	}
 
-	// Assumption: Subscription CR and Subscriber should be deployed in the same namespace
+	// Assumption: Subscription CR and Subscriber should be deployed in the same namespace.
```

Comment:
Had to modify something so the `pre-kyma-components-eventing-controller` job gets triggered! Otherwise, the other jobs would fail since no PR-image is produced.